docs/reference/searchable-snapshots/apis/mount-snapshot.asciidoc

```diff
@@ -23,6 +23,8 @@ For more information, see <<security-privileges>>.
 [[searchable-snapshots-api-mount-desc]]
 ==== {api-description-title}
 
+This API mounts a snapshot as a searchable snapshot index. 
+Note that manually mounting {ilm-init}-managed snapshots can <<manually-mounting-snapshots,interfere>> with <<index-lifecycle-management,{ilm-init} processes>>.
```

Comment:
I'd drop this in as a separate paragraph so it's more visible. Also, avoid "note that" - either it should be a callout, or it's just a sentence. technically all documentation is just noting things.

I think that because this can cause problems, and we want to discourage people from using this API in the context of ILM-managed snapshots, I'd say that a little more clearly as well.

```suggestion

Don't use this API for snapshots managed by {ilm-init}. Manually mounting {ilm-init}-managed snapshots can <<manually-mounting-snapshots,interfere>> with <<index-lifecycle-management,{ilm-init} processes>>.
```